.gitmodules

```diff
@@ -3,7 +3,7 @@
   #url = https://github.com/spack/spack
   #branch = develop
   url = https://github.com/jcsda/spack
-  branch = jcsda_emc_spack_stack
+  branch = JCSDA/spack@24657b6
```

Comment:
This change needs to be reverted. You need to update the submodule pointer, not `.gitmodules`.

Reply:
@climbfuji Can you point me to the correct file and/or give me explicit instructions? I don't know what the "submodule pointer" is referring to.

Reply:
In your local spack-stack clone of your branch `subfeature/packages_py`:
```
cd spack
git remote update
git checkout origin/jcsda_emc_spack_stack
git log
# this should show you the hash I mentioned at the very top
cd ..
git diff
# you should see a diff showing up for "spack (new commits)"
git add spack
git commit -m "Update submodule pointer for spack"
git push YOUR_REMOTE_NAME subfeature/packages_py
```

Reply:
@climbfuji OK. I just submitted attempt number 2.

Reply:
submodule pointer is looking good now, just need to revert `.gitmodules`.

Reply:
To save headaches, how should I properly do that given our current state of the repo?

Reply:
@climbfuji OK, reverted. 

Thank you for your patience with me.

Reply:
CI tests are running now, thanks!

Reply:
They shouldn't be affected by your changes, but I'll let them finish nonetheless before merging.